service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageDocumentsControllerAboutToSubmitTest.java

```diff
@@ -52,6 +59,14 @@
     @Autowired
     private Time time;
 
+    @MockBean
+    private IdamClient idamClient;
+
+    @BeforeEach
+    void init() {
+        given(idamClient.getUserDetails(eq(USER_AUTH_TOKEN))).willReturn(createUserDetailsWithHmctsRole());
+    }
```

Comment:
Looks like we are test to make sure SupportingEvidenceBundles are updated with `author`.

Reply:
added